content/faqs/html-lazy-not-working.md

```diff
@@ -0,0 +1,112 @@
+---
+type: page
+title: Why is my use of Html.Lazy not working?
+description: We explore some common gotcha's for Html.Lazy
+published: true
+status: seedling
+---
+
+The [Elm guide on Html.Lazy](https://guide.elm-lang.org/optimization/lazy.html) is the best place to start.
+
+Here are some additional gotcha's that are helped by examples.
+
+
+### Understand what consitutes a changed value
+
+The Elm guide has the following note:
+
+> Note: When are two values “the same” though? To optimize for performance, we use JavaScript’s `===` operator behind the scenes:
+>
+> Structural equality is used for `Int`, `Float`, `String`, `Char`, and `Bool`.
+> Reference equality is used for records, lists, custom types, dictionaries, etc.
+> Structural equality means that `4` is the same as `4` no matter how you produced those values. Reference equality means the actual pointer in memory has to be the same. Using reference equality is always cheap `O(1)`, even when the data structure has thousands or millions of entries. So this is mostly about making sure that using `lazy` will never slow your code down a bunch by accident. All the checks are super cheap!
+
+Sometimes it's not clear however when we're ending up with new references. Here are a few common pitfalls:
+
+
+### Gotcha: New values created in the view
```

Comment:
```suggestion
### Gotcha: New references created for the arguments
```

Not entirely sure about this either, but the "in the view" is a bit confusing :thinking: